documentation/src/main/resources/_posts/2022-10-30-live-message-conditions.md

```diff
@@ -0,0 +1,162 @@
+
+If the given condition is invalid, the request will fail with HTTP status code **400 - Bad Request**.
+
+More documentation for this feature can be found here: [Conditional Requests](basic-conditional-requests.html)
+
+### Permissions for conditional requests
+
+In order to execute a conditional request, the authorized subject needs to have WRITE permission at the resource
+that should be changed by the request.
+
+Additionally, the authorized subject needs to have READ permission at the resource used in the specified condition.
+Given the condition from the introduction `condition=eq(attributes/location,"Wonderland")`,
+read access on the single attribute would be sufficient.
+However, the condition can also be more complex, or include other sub-structures of the thing.
+Then of course, the authorized subject needs READ permission on all parameters of the specified condition.
+
+## Examples
+The following sub-sections will show how to use conditional requests via the HTTP API, the Ditto protocol,
+and the Ditto Java Client.
+
+To demonstrate the new conditional request, we assume that the following thing already exists:
+
+```json
+{
+  "thingId": "org.eclipse.ditto:carbon-monoxide-alarm",
+  "policyId": "org.eclipse.ditto:carbon-monoxide-alarm",
+  "attributes": {
+    "manufacturer": "ACME demo corp.",
+    "location": "Wonderland",
+    "serialno": "42"
+  },
+  "features": {
+    "carbon-monoxide-level": {
+      "properties": {
+        "ppm,": 2
+      }
+    },
+    "alarm": {
+      "properties": {
+        "lastTriggered": "2021-09-23T07:01:56Z",
+        "confirmed": false
+        }
+      }
+    }
+  }
+}
+```
+
+### Condition based on alarm/confirmed
+In this example a live alarm message from the device should only be sent, if the alarm confirmed property is set to 
+false by the end user application. This is done to prevent duplicate received alarms by the customer.
+
+Another use case could be to i.e. only send a message to a device when the device is connected:
+```
+POST /api/2/things/org.eclipse.ditto:my-thing-1/inbox/messages/doSomething?condition=gt(features/ConnectionStatus/properties/status/readyUntil,time:now)
+```
+
+### Permissions to execute the example
+For this example, the authorized subject could have READ and WRITE permissions on the complete thing resource.
+However, it is only necessary on the path _thing:/features/alarm/properties/confirmed_.
+
+## Conditional requests via HTTP API
+Using the HTTP API the condition can either be specified via HTTP Header or via HTTP query parameter.  
+In this section, we will show how to use both options.
+
+### Conditional request with HTTP Header
+```
+curl -X PATCH -H 'Content-Type: application/json' -H 'condition: eq(features/alarm/properties/confirmed/,false)' /api/2/things/org.eclipse.ditto:carbon-monoxide-alarm/outbox/messages/co-alarm -d '{ "CO Level to high! Open your windows!" }'
```

Comment:
PATCH? Shouldn't sending a message use `POST`?

The host and port from the URL is also missing - did you try that example out?